x-pack/plugins/alerting/common/alert_summary.ts

```diff
@@ -40,4 +40,5 @@ export interface AlertStatus {
   activeStartDate?: string;
   flapping: boolean;
   maintenanceWindowIds?: string[];
+  tracked: boolean;
```

Comment:
tell if this is stupid but should we have it the other way and just know how many alert who are `unTracked`